### ISSUE TITLE
UniFI app does not show any statistics; test button shows invalid credentials

### ISSUE THREAD

**Issue by gitterdoneplease** [linuxserver/Heimdall-Apps#695]
This seems to be somewhat known. The UniFi app doesn't seem to be supported correctly.

**To Reproduce**
1. Add app with type Unifi
2. Fill out specified information
3. click test
4. "Invalid Credentials!" is result

**Expected behavior**
This should work in the test and then after saved it should show the statistics.

**Screenshots**
<img width="312" alt="Screenshot 2024-02-22 at 17 34 00" src="https://github.com/linuxserver/Heimdall-Apps/assets/20113274/5745286f-a1fe-494d-a6bb-a8714ee3d86e">

**Version info (please complete the following information):**
 - Heimdall:  2.6.1
 - Docker, if applicable: N/A
 - App: N/A
 - Version of remote application App tries to use: 8.0.26

**Additional context**
This is a really great feature that I would love to use.

**Comment by mvdkleijn** [comment]
You're not wrong. Since I came onboard as maintainer I found several "enhanced" apps that did nothing.

Unfortunately I don't have anything related to Unifi so not sure how much I can help here, at least for now.

**Comment by mvdkleijn** [comment]
@gitterdoneplease if you could get it into a PR, that'd be nice... I have time to look at PRs etc but I don't really have time to work on all the different apps.

**Comment by zaepho** [comment]
see PR #798 for Self Hosted controllers with this issues after the API URL switch in #626
